src/reporting/reporting.md

```diff
@@ -0,0 +1,52 @@
+# Overview of Reporting
+
+Reporting is a Toca user's hub for anything related to the jobs which are being run at any given time. A user can have multiple reports to accommodate for different queries and dashboards. The current report is denoted in the top left alongside dropdown options to switch reports or create a new one.
+
+**Jobs page** - The jobs page is the default home screen for reporting. Here you will see every job which your user has the permissions to see, (at least 'read'), and which adheres to the current query. New jobs which have just been run will appear at the top of the feed.
+
+**Columns**
+- Name - The name of the job.
+- Duration - How long the job took to run.
+- Queued - When the user queued the job.
+- Started - When the job began to run.
+- Finished - The time when the job finished.
+- Triggered by - the method via which the job was triggered e.g via schedule, listener, app user etc.
+- Publish state - The version of the job which was run. This could be either: draft, published or previous.
+- Bots - Hovering over either the monitor or cloud icons will show you which bot the job ran on and the corresponding bot version. This is useful for debugging.
+
+Many of the columns in the jobs page can be sorted by clicking on the column header.
+
+ **Further Options**
+- Open workflow - Opens the workflow designer in a separate tab.
+- Open project - Opens the project that the workflow belongs to in a separate tab.
+- Rerun job - This will rerun the job with the exact same inputs etc as it was previously run.
+
+**Queries** -  The jobs which appear in the jobs page are those which adhere to the current query. This query can be seen by pressing the query button in the top right hand corner. The query fields are as such:
+- Field - A selection of categories to filter on, many of which are represented via the columns in the jobs page.
+- Operator - This will depend on the field chosen as part of that query line. Many fields will give the operator 'equals' or 'doesn't equal', whereas those which relate to time may show 'before' or 'after'.
+- Value - This field will often show a dropdown list of available options based on which field has been chosen as part of that query line. Other examples include a separate modal popup if the field relates to time.
```

Comment:
Better would be to say something like `The input will be appropriate for the selected field - values will be suggest where possible and date/time fields will use a custom selector`.